### ISSUE TITLE
Use of absolute scale don't work

### ISSUE THREAD

**Issue by zanac** [kandsten/ha-heatmap-card#12]
i'm trying this syntax and it return: Cannot assign to read only property 'value' of object '#<Object>'

type: custom:heatmap-card
entity: sensor.ewelink_th01_temperature_2
scale:
  type: absolute
  steps:
    - value: -10
      color: '#000000'
    - value: 20
      color: '#FFFF00'
    - value: 40
      color: '#FF00FF'

**Comment by kandsten** [comment]
Right, managed to slip in a regression, thanks for the bug report. I have a fix for it, will push soon'ish.

**Comment by L0bit0** [comment]
Any update on this? I was crazy today reviewing my yaml. :-)

I assume i got the same issue:
![image](https://github.com/kandsten/ha-heatmap-card/assets/10359623/73d1f08c-7f38-477b-85af-807354616d4d)
NOTE: i had same issue with relative or absolute...

**Comment by L0bit0** [comment]
Hi there is any update on this? 

thanks in advance

**Comment by aid7c1** [comment]
me too same issue

**Comment by zanac** [comment]
> me too same issue

if you want i have made an extension for temperature https://github.com/zanac/temperature-heatmap-card if you want to try this is a "variant" of the good work of this
